notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            rad_dist_lat, rad_dist_lon = np.radians(pos2[0] - pos1[0]), np.radians(
</pre><p>For readability can you break these up and calculate <code>rad_dist_lat</code> and <code>rad_dist_lon</code> in separate statements?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>